runtime/mainnet/src/config/monetary.rs

```diff
@@ -0,0 +1,439 @@
+use pop_runtime_common::UNIT;
+use sp_runtime::traits::AccountIdConversion;
+
+use crate::{
+	parameter_types, AccountId, Balance, Balances, ConstU32, ConstU8, ConstantMultiplier, PalletId,
+	ResolveTo, Runtime, RuntimeEvent, RuntimeFreezeReason, RuntimeHoldReason,
+	SlowAdjustingFeeUpdate, System, VariantCountOf, EXISTENTIAL_DEPOSIT,
+};
+
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
+	// src: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/constants/src/polkadot.rs#L70
+	(items as Balance * 20 * UNIT + (bytes as Balance) * 100 * fee::MILLI_CENTS) / 100
+}
+
+/// Constants related to Polkadot fee payment.
+/// Source: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/constants/src/polkadot.rs#L65C47-L65C58
```

Comment:
Doesn't have to be included in docs tmo, serves more as a reference for code maintainers / readers:
```suggestion
// Source: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/constants/src/polkadot.rs#L65C47-L65C58
```

Reply:
[e0a32be](https://github.com/r0gue-io/pop-node/pull/440/commits/e0a32be19982f4cf62aeaa11cb264a9959d2c3ef)